tests/integration-tests/tests/client_tests.rs

```diff
@@ -164,7 +164,7 @@ where
     .set_max_payload(1232)
     .set_version(0);
 
-    let response = client.send(msg).expect("Query failed");
+    let response = client.send(msg).remove(0).expect("Query failed");
```

Comment:
no `first_answer` here?

Reply:
it's a sync client, so no future/stream, but a Vec directly